Sources/Transcripts/Episodes/0000-Introduction.swift

```diff
@@ -32,85 +32,6 @@ extension Episode {
       ),
       vimeoId: 354_215_017
     ),
-    transcriptBlocks: _privateTranscriptBlocks
+    transcriptBlocks: loadTranscriptBlocks(forSequence: 0)
```

Comment:
this is my hack for right now to load transcripts from a resource. I think eventually we will want to lazily load these on first access.